logdna/resource_view_test.go

```diff
@@ -302,8 +302,8 @@ func TestViewJSONUpdateError(t *testing.T) {
 	levels2 := "critical"
 	host1 := "host1"
 	host2 := "host2"
-	category1 := "DEMOCATEGORY1"
-	category2 := "DemoCategory2"
+	category1 := "category1"
+	category2 := "category2"
```

Comment:
changing these will require changes on the integration account in production - is there a reason for switching these? 

also iirc these were set up with mixed casing specifically for testing that scenario

Reply:
@mdeltito nah this was just a byproduct of troubleshooting the tests, I was using the service key for the `LogDNA Canary` account in production instead of the `MDT-terraform` account which is why the Jenkins builds weren't passing. This took me an unfortunate amount of time to figure out since the local testing didn't 100% align with the build results.

Reverted my changes since the tests were already correct initially.

Reply:
> instead of the MDT-terraform account

@david-ly the CI account is named: 

```
Terraform Plugin Provider Tests (terraform-provider-logdna)
```

we should get that into the docs and make sure it's available to everyone

Reply:
Access control is wide open, I've confirmed.  Looks like just a doc issue now.
```
Allow members with an @logdna.com email to join this organization without a request.
```

Reply:
Gotcha, thanks for clarifying. I'll add this to the docs in a future PR